mesh/mesh_connections.hpp

```diff
@@ -0,0 +1,112 @@
+class MeshConnections
+{
+   private:
+   Mesh *mesh;
+   Table *elem_to_elem, *elem_to_face, *elem_to_edge, *elem_to vert;
+   Table *face_to_elem, *face_to_face, *face_to_edge, *face_to vert;
+   Table *edge_to_elem, *edge_to_face, *edge_to_edge, *edge_to vert;
+   Table *vert_to_elem, *vert_to_face, *vert_to_edge, *vert_to vert;
+
+   public:
+   struct EntityID
+   {
+      int dim;
+      int id;
+      bool boundary;
+   }
+
+   struct EntityIDs
+   {
+      int dim;
+      Array<int> ids;
+      bool boundary;
+   }
+
+   //Dimension independent interface
```

Comment:
Hopefully converging on the general interface.  The question remains, do we want the call signatures of the named methods in addition to these.

Reply:
Any more comments on this interface?  If not, I would like to start writing toward it.